### PR TITLE
create terraform-provider ui

### PR DESCRIPTION
**Pull request by mageexcel**

### CODE REVIEW

File: components/terraform/Header.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="hero header row justify-content-center">
+        <div class="col-12 col-md-12 col-lg-10">
+            <div class="row">
+                <div class="col-12 col-md-6">
+                    <h1>Kestra Terraform Provider: Deploy All Your Workflows in a CI/CD Process</h1>
+                    <p>Manage and deploy Kestra workflows directly within your existing Terraform environment, eliminating the need for separate configurations.</p>
```
Comment (anna-geller) [inline components/terraform/Header.vue:7]:
```suggestion
                    <h1>Kestra Terraform Provider: Deploy All Your Workflows <br/> in a CI/CD Process</h1>
                    <p>Manage and deploy Kestra workflows directly within your existing Terraform environment, eliminating the need for separate configurations.</p>
```
there must be a line break before "in a CI/CD Process"

File: components/terraform/Overview.vue [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                        <p>You can manage and deploy Kestra workflows directly within your existing Terraform environment, eliminating the need for separate configurations.</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-sm-6 col-md-4 col-lg-3 mb-4">
+                <div class="card h-100 d-flex">
+                    <div class="card-body text-center">
+                        <img src="/terraform/overview/hub.svg" alt="hub svg">
+                        <h6>Centralized Management</h6>
+                        <p>Leveraging the Kestra Terraform Provider allows you to manage your Kestra resources along with other infrastructural components, all from a single platform.</p>
```
Comment (anna-geller) [inline components/terraform/Overview.vue:18]:
```suggestion
                        <p>The Terraform Provider allows you to manage your Kestra resources in code, along with other infrastructure components, all from a single platform.</p>
```

File: components/terraform/Overview.vue [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-sm-6 col-md-4 col-lg-3 mb-4">
+                <div class="card h-100 d-flex">
+                    <div class="card-body text-center">
+                        <img src="/terraform/overview/code-box-parentheses.svg" alt="code-box-parentheses svg">
+                        <h6>Infrastructure as Code</h6>
+                        <p>By using Terraform, you benefit from all the Infrastructure as Code (IaC) best practices, making it easier to version control, collaborate, and maintain your Kestra configurations.</p>
```
Comment (anna-geller) [inline components/terraform/Overview.vue:27]:
```suggestion
                        <p>By using Terraform, you benefit from the Infrastructure as Code (IaC) best practices, making it easier to version control, collaborate, and maintain your Kestra configurations.</p>
```

File: components/terraform/Overview.vue [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-sm-6 col-md-4 col-lg-3 mb-4">
+                <div class="card h-100 d-flex">
+                    <div class="card-body text-center">
+                        <img src="/terraform/overview/api.svg" alt="api svg">
+                        <h6>API-First Approach</h6>
+                        <p>The provider is built on Kestra's API-first architecture, ensuring seamless integration and greater flexibility in managing your data workflows.</p>
```
Comment (anna-geller) [inline components/terraform/Overview.vue:36]:
```suggestion
                        <p>The provider is built on top of Kestra's API, allowing you to programmatically manage all resources, from workflows, namespaces, and secrets, to user management.</p>
```

File: components/terraform/Overview.vue [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+            </div>
+        </div>
+        <div class="hero row justify-content-center align-items-center">
+            <div class="col-12 col-md-6">
+                <img src="/terraform/overview/declarative-configurations-code.svg" class="w-100" alt="declarative configurations code svg">
+            </div>
+            <div class="col-12 col-md-6">
+                <div class="row justify-content-center">
+                    <div class="col-12 col-md-7">
+                        <h3>Declarative Configurations</h3>
```
Comment (anna-geller) [inline components/terraform/Overview.vue:48]:
```suggestion
                        <h3>Declarative Configuration</h3>
```

File: components/terraform/Overview.vue [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+                        <p>Terraform uses its HashiCorp Configuration Language (HCL), while Kestra flows are defined in YAML. This alignment in philosophy ensures not only a smooth transition but also a lower learning curve.</p>
```
Comment (anna-geller) [inline components/terraform/Overview.vue:49]:
```suggestion
                        <p>Terraform uses its HashiCorp Configuration Language (HCL), while Kestra flows are defined in YAML. This alignment in declarative interfaces ensures not only a smooth transition but also makes it easy to use both technologies together.</p>
```

File: components/terraform/Overview.vue [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-md-6">
+                <div class="row justify-content-center">
+                    <div class="col-12 col-md-7">
+                        <h3>Comprehensive & Modular Resource Management</h3>
```
Comment (anna-geller) [inline components/terraform/Overview.vue:56]:
```suggestion
                        <h3>Modular Resource Management</h3>
```

File: components/terraform/Overview.vue [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+                        <p>Our Terraform provider for Kestra is designed to easily manage your Kestra resources using Terraform. This includes, among others, flows, namespace, and user management, as well as secrets. Check the list of available resources <NuxtLink href="/docs/terraform">in our documentation.</NuxtLink></p>
```
Comment (anna-geller) [inline components/terraform/Overview.vue:57]:
```suggestion
                        <p>Kestra's Terraform provider is designed to easily manage your resources using Terraform. This includes, among others, flows, namespace, and user management, as well as secrets. Check the list of available resources in our <NuxtLink href="/docs/terraform">documentation.</NuxtLink></p>
```

File: components/terraform/Overview.vue [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-md-6">
+                <img src="/terraform/overview/modular-resource-management-code.svg" class="w-100" alt="modular resource management code svg">
+            </div>
+            <div class="col-12 col-md-6">
+                <p class="mt-5 text-white">Additionally, the inherent modularity in both Terraform and Kestra's configurations allows you to create reusable blocks of code. These can be shared across multiple teams or projects, contributing to more efficient, maintainable operations.</p>
```
Comment (anna-geller) [inline components/terraform/Overview.vue:65]:
```suggestion
                <p class="mt-5 text-white">The inherent modularity in both, Terraform and Kestra, allows you to create reusable blocks of code. These can be shared across multiple teams or projects, contributing to maintainable operations.</p>
```

File: components/terraform/UseCases.vue [hunk 10]
```diff
@@ -0,0 +1,39 @@
+<template>
+    <LayoutSection title="Use Cases">
```
Comment (anna-geller) [inline components/terraform/UseCases.vue:2]:
```suggestion
    <LayoutSection title="Benefits of the Integration">
```

File: components/terraform/Footer.vue [hunk 11]
```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="row justify-content-center">
+        <div class="col-12 col-md-7">
+            <div class="container text-center mb-5 rounded">
+                <div>
+                    <h3 data-aos="fade-right">Getting Started</h3>
+                    <h6 class="mt-3" data-aos="zoom-in">
+                        <p>Integration between Kestra and Terraform is straightforward. Add the Kestra provider to your Terraform main configuration file to get started.</p>
```
Comment (anna-geller) [inline components/terraform/Footer.vue:8]:
```suggestion
                        <p>Integration between Kestra and Terraform is simple. Add the Kestra provider to your Terraform configuration file to get started.</p>
```

**Comment by Skraye** [comment]
UI changes
Benefits of integration, images have strange background instead of just dot :
image
Overview, Missing halo effect around image (first website, second figma): 
image
image

File: components/terraform/Footer.vue [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+        </div>
+    </div>
+</template>
+    
+<style scoped lang="scss">
+    @import "../../assets/styles/variable";
+    
+    .container {     
+        position: relative;
+        background: #2A1940;
```
Comment (Skraye) [inline components/terraform/Footer.vue:29]: Add a variable, create one if necessary, so we can list all colors used in website

File: components/terraform/Header.vue [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+.header {
+    background: url('terraform/header/background.svg') no-repeat center;
+
+    p {
+        font-size: $font-size-xl;
+    }
+
+    .docs-btn {
+        border: double 1px transparent;
+        background-image: linear-gradient(#2A1940, #2A1940), radial-gradient(circle at top left, #B010FB 18.11%, #DE97FF 27.65%, #A227DB 36.31%, rgba(162, 39, 219, 0) 58.84%, rgba(166, 16, 236, 0.84) 80.5%);
```
Comment (Skraye) [inline components/terraform/Header.vue:33]: using existing class instead :  btn-animated btn-purple-animated

File: components/terraform/Overview.vue [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+        padding: 2rem 1rem;
+
+        h6 {
+            line-height: 1.875rem;
+        }
+
+        p {
+            font-size: $font-size-sm;
+            line-height: 1.375rem;
+            color: #CFCEFF;
```
Comment (Skraye) [inline components/terraform/Overview.vue:92]: Add a variable, create one if necessary, so we can list all colors used in website

File: components/terraform/Overview.vue [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+</template>
+
+<style scoped lang="scss">
+@import "../../assets/styles/variable";
+.overview {
+    background: url('/terraform/overview/background.svg') no-repeat 0 -4%;
+}
+
+.card {
+    background-color: #110221;
```
Comment (Skraye) [inline components/terraform/Overview.vue:78]: Add a variable, create one if necessary, so we can list all colors used in website

File: components/terraform/Overview.vue [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+    }
+
+}
+
+.hero {
+    padding-bottom: 0;
+
+    p {
+        line-height: 1.5rem;
+        color: #CFCEFF;
```
Comment (Skraye) [inline components/terraform/Overview.vue:103]: Add a variable, create one if necessary, so we can list all colors used in website

File: pages/terraform-provider.vue [hunk 17]
```diff
@@ -0,0 +1,17 @@
...
+            <TerraformOverview />
+            <TerraformUseCases />
+            <TerraformFooter />
+        </div>
+    </div>
+</template>
+
+<style scoped lang="scss">
+.main {
+    background-color: #0F011D;
```
Comment (Skraye) [inline pages/terraform-provider.vue:14]: Add a variable, create one if necessary, so we can list all colors used in website

**Review by Skraye — CHANGES_REQUESTED** [review]

**Pull request merged**